pyaedt/icepak.py

```diff
@@ -177,6 +177,12 @@ def assign_2way_coupling(self,setup_name=None, number_of_iterations=2, continue_
         bool
             ``True`` when successful, ``False`` when failed.
 
+        Examples
+        --------
+
```

Comment:
It would be nice to have short descriptions for these examples.  If you feel they are straightforward enough to not need it, then please ignore.

Reply:
I have added descriptions to the more complex examples and left the simple ones as is.